trieste/acquisition/optimizer.py

```diff
@@ -98,32 +100,54 @@ def automatic_optimizer_selector(
 
     else:
         raise NotImplementedError(
-            f""" No optimizer currentely supports acquisition function
+            f""" No optimizer currently supports acquisition function
                     maximisation over search spaces of type {space}.
                     Try specifying the optimize_random optimizer"""
         )
 
 
-def optimize_discrete(space: DiscreteSearchSpace, target_func: AcquisitionFunction) -> TensorType:
+def optimize_discrete(
+    space: DiscreteSearchSpace,
+    target_func: Union[AcquisitionFunction, Tuple[AcquisitionFunction, int]],
+) -> TensorType:
     """
     An :const:`AcquisitionOptimizer` for :class:'DiscreteSearchSpace' spaces and
     batches of size of 1.
 
+    This function also support the maximization of vectorized target functions by evaluating
+    the points in the search space for each of the individual V functions making up target_func.
+
     :param space: The space of points over which to search, for points with shape [D].
     :param target_func: The function to maximise, with input shape [..., 1, D] and output shape
             [..., 1].
     :return: The **one** point in ``space`` that maximises ``target_func``, with shape [1, D].
     """
-    target_func_values = target_func(space.points[:, None, :])
+
```

Comment:
probably it would be useful to check before this that target_func is either an instance of AcquisitionFunction or a tuple of AcquisitionFunction and int